examples/cgo/cc_dependency/BUILD

```diff
@@ -0,0 +1,36 @@
+config_setting(
+    name = "darwin",
+    values = {"cpu": "darwin"},
+)
+
+cc_library(
+    name = "version",
+    srcs = ["cxx_version.cc"],
+    hdrs = ["version.h"],
+    # TODO(yugui) Support darwin too and remove this workaround.
+    # See also comments in cxx_version.cc.
```

Comment:
There is actually an issue in fixing it on OSX: you don't see a rpath option, you said where to find the library. I guess I need to spend more than 5 minutes to make it work because we have to figure out which is the good rpath for every passed library.

/cc @lberki that's another hell in our CROSSTOOL file.